scripts/tests/api_compare/docker-compose.yml

```diff
@@ -2,70 +2,186 @@
       - filecoin-proofs:${FIL_PROOFS_PARAMETER_CACHE}
     networks:
       - api-tests
     environment:
       - FIL_PROOFS_PARAMETER_CACHE=${FIL_PROOFS_PARAMETER_CACHE}
       - LOTUS_API_LISTENADDRESS=/ip4/0.0.0.0/tcp/${LOTUS_RPC_PORT}/http
       - LOTUS_FEVM_ENABLEETHRPC=1
-    entrypoint: ["/bin/bash", "-c" ]
+      - LOTUS_CHAINSTORE_ENABLESPLITSTORE=false
+      - FULLNODE_API_INFO=/dns/lotus/tcp/${LOTUS_RPC_PORT}/http
+    entrypoint: [ "/bin/bash", "-c" ]
+    command:
+      - |
+        set -euxo pipefail
+        lotus daemon --import-snapshot "$SNAPSHOT_PATH" --remove-existing-chain
+  lotus-sync-wait:
+    depends_on:
+      lotus:
+        condition: service_started
+    image: ${LOTUS_IMAGE}
+    volumes:
+      - node-data:/data
+      - lotus-data:/var/lib/lotus
+      - filecoin-proofs:${FIL_PROOFS_PARAMETER_CACHE}
+    networks:
+      - api-tests
+    environment:
+      - FULLNODE_API_INFO=/dns/lotus/tcp/${LOTUS_RPC_PORT}/http
+    entrypoint: [ "/bin/bash", "-c" ]
+    command:
+      - |
+        set -euxo pipefail
+        until lotus sync wait; do
+            sleep 5
+        done
+  api-compare:
+    depends_on:
+      lotus-sync-wait:
+        condition: service_completed_successfully
+    build:
+      context: ../../../.
+      dockerfile: ${FOREST_DOCKERFILE_OVERRIDE:-Dockerfile}
+    volumes:
+      - node-data:/data
+      - filecoin-proofs:${FIL_PROOFS_PARAMETER_CACHE}
+      - ./filter-list:/data/filter-list
+    networks:
+      - api-tests
+    entrypoint: [ "/bin/bash", "-c" ]
+    command:
+      - |
+        set -euxo pipefail
+        forest-tool api compare "$SNAPSHOT_PATH" \
+          --forest "/dns/forest/tcp/$FOREST_RPC_PORT/http" \
+          --lotus "/dns/lotus/tcp/$LOTUS_RPC_PORT/http" \
+          --n-tipsets 5 \
+          --filter-file /data/filter-list
+  api-compare-offline:
+    depends_on:
+      lotus-sync-wait:
+        condition: service_completed_successfully
+    build:
+      context: ../../../.
+      dockerfile: ${FOREST_DOCKERFILE_OVERRIDE:-Dockerfile}
+    volumes:
+      - node-data:/data
+      - filecoin-proofs:${FIL_PROOFS_PARAMETER_CACHE}
+      - ./filter-list:/data/filter-list
+    networks:
+      - api-tests
+    entrypoint: [ "/bin/bash", "-c" ]
+    command:
+      - |
+        set -euxo pipefail
+        forest-tool api compare "$SNAPSHOT_PATH" \
+          --forest "/dns/api-serve/tcp/$FOREST_OFFLINE_RPC_PORT/http" \
+          --lotus "/dns/lotus/tcp/$LOTUS_RPC_PORT/http" \
+          --n-tipsets 5 \
+          --filter-file /data/filter-list || true # Fixme
+  post-setup:
+    depends_on:
+      api-compare:
+        required: true
+        condition: service_completed_successfully
+      api-compare-offline:
+        required: true
+        condition: service_completed_successfully
+    image: ${LOTUS_IMAGE}
+    volumes:
+      - node-data:/data
+      - lotus-data:/var/lib/lotus
+      - filecoin-proofs:${FIL_PROOFS_PARAMETER_CACHE}
```

Comment:
Are those volumes needed?

Reply:
Lotus Dockerfile defines these volumes, if not mounted random volumes are created

Reply:
But I guess I can use the mostly pulled `alpine` or `busybox` image or this

Reply:
Hmm, unable to pull `busybox`

```
Unable to find image 'buzybox:latest' locally
docker: Error response from daemon: pull access denied for buzybox, repository does not exist or may require 'docker login': denied: requested access to the resource is denied.
```

Reply:
Ouch, good catch